product_sale_manufactured_for/README.rst

```diff
@@ -0,0 +1,87 @@
+=======================
+Sale Commercial Partner
```

Comment:
are you sure it is correct module name here?

Reply:
No is wrong. The name should be "Product Manufactured for Customer"